kots-preflight.yaml

```diff
@@ -53,7 +53,7 @@ spec:
           - warn:
               message: Unable to determine the distribution of Kubernetes.
     - nodeResources:
-        checkName: Total CPU Cores
+        checkName: Required Total CPU Cores
         outcomes:
           - fail:
               when: "sum(cpuCapacity) < 2"
```

Comment:
Is this correct? Shouldn't it be ` < 4`

Reply:
How come "required" if this isn't a strict preflight check? Required is the wording we put in the admin console on a strict preflight check, so saying "required" here is confusing IMO. I didn't mind the previous wording where it just said "Total CPU Cores".

Josh is right that it should be < 4. That's what the `message` field says too:
>The cluster should contain at least 4 cores.